client/blocks/simple-site-rename-form/style.scss

```diff
@@ -0,0 +1,98 @@
+/** @format */
+
+.simple-site-rename-form__dialog {
+	text-align: left;
+	max-width: 400px;
```

Comment:
Random thought:
This px value feels a bit arbitrary. We should store the used width values for dialogs as variables somewhere. Or use default widths that come with the dialog component. Otherwise, with time users will encounter dialogs that are all over the place in terms of size. But that's a discussion outside the scope of this PR I guess.

Reply:
Very much agree. Naturally we will need modals of different sizes for different content (media-modal is large for example) but we could well benefit from having a set number of different sizes - small, medium large defaults could be useful :)